src/plugins/kibana/public/dashboard/index.js

```diff
@@ -105,7 +105,9 @@ app.directive('dashboardApp', function (Notifier, courier, AppState, timefilter,
         load: require('plugins/kibana/dashboard/partials/load_dashboard.html'),
         share: require('plugins/kibana/dashboard/partials/share.html'),
         pickVis: require('plugins/kibana/dashboard/partials/pick_visualization.html'),
-        options: require('plugins/kibana/dashboard/partials/options.html')
+        options: require('plugins/kibana/dashboard/partials/options.html'),
```

Comment:
There are a few require statements throughout the code bse as you can see above, I say we open an issue to get to all of them.